src/askvm/resources/core/__test__/query.test.ts

```diff
@@ -0,0 +1,93 @@
+import { runUntyped } from '../../..';
+import { parse } from '../../../../askcode';
```

Comment:
I wonder if for resource unit test we should be calling the resource directly or keep going through the AskCode parser like you're doing here 🤔 . Maybe it would be easier to catch any potential problems and check for different scenarios if we were calling `query` resource directly. What do you guys think @YonatanKra @czerwinskilukasz1?

Reply:
I didn't even notice it. Just my IDE auto import doing its thing.
Definitely better to do direct here. The post build tests willmake sure it is available in the barrel file.

Reply:
Looking closer at your response I now see you mean to test the function directly. 
Should make for easier testing.

I'll try that.

I just took existing tests and followed conventions in this area.

Both ways work for me as long as I can test. 

Main point is to decide on good conventions and stick to them so read8ng one part of code would look similar to others (in regards to style etc.).

Reply:
@YonatanKra yes, you are right that we're missing consistency there. I think personally that low level unit test for a function should just test this function and not rely to much on other code - this way they are more resilient and allow for testing a greater variety of edge cases. We also have this other kind of tests that are more like "unit integration tests" where we checking results for askScript programs.

Either way both approaches here are acceptable.